substrate/frame/support/test/tests/pallet_ui/pass/feeless_call.rs

```diff
@@ -0,0 +1,37 @@
+// This file is part of Substrate.
```

Comment:
Should/is this be a UI test if it doesn't check stderr?

Reply:
This UI test is actually tested as a success case. The assertion here is that it should compile successfully.